lib/qb_integration/services/customer.rb

```diff
@@ -7,6 +7,43 @@ def initialize(config, payload)
         super("Customer", config)
 
         @order = payload[:order] || payload[:invoice] || {}
+        @customer = payload[:customer]
+      end
+
+      def create_customer
+        if @customer[:qbo_id] && !!@config.fetch("create_or_update")
```

Comment:
`@config.fetch("create_or_update")` is a param that is used only on the /update_customer endpoint. If true, it's saying "you're allowed to create a new customer if one doesn't exist in QBO".
When using the /add_customer endpoint, you should _always_ add a new customer if one doesn't exists AND should always update a customer if one is found.

Reply:
Ahh, ok. Thanks for clarifying that!